### PR TITLE
chore(dbg): allow distro specific driverkit configs generation.

### PR DESCRIPTION
**Pull request by FedeDP**
This leverage new `last_run_distro` file present in kernels branch of kernel-crawler.

See https://github.com/falcosecurity/kernel-crawler/pull/151 and https://github.com/falcosecurity/kernel-crawler/pull/150

### CODE REVIEW

File: driverkit/utils/scrape_and_generate [hunk 1]
```diff
@@ -6,18 +6,35 @@ DRY_RUN="${DRY_RUN:-false}"
...
+LASTRUN_DISTRO="*"
 
 function pretty_echo() {
 	echo
 	echo "> ${1}"
 }
 
 function get_kernel_releases() {
-	curl -sL -o $TMPDIR/sample.json $LIST_URL
+	curl -sL -o $TMPDIR/list.json $LIST_URL
```
Comment (FedeDP) [inline driverkit/utils/scrape_and_generate:18]: renamed to `list.json`

**Review by cappellinsamuele — APPROVED** [review]

**Pull request merged**